test/Microsoft.AspNet.Localization.Tests/AccessLanguageHeaderRequestCultureProviderTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved. 
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Builder;
+using Microsoft.AspNet.Localization;
+using Microsoft.AspNet.TestHost;
+using Microsoft.Framework.DependencyInjection;
+using Xunit;
+
+namespace Microsoft.Framework.Localization.Tests
+{
+    public class AccessLanguageHeaderRequestCultureProviderTest
+    {
+        [Fact]
+        public async void GetFallbackLanguage()
+        {
+            using (var server = TestServer.Create(app =>
+            {
+                var options = new RequestLocalizationOptions
+                {
+                    SupportedCultures = new List<CultureInfo>
+                    {
+                        new CultureInfo("ar-SA"),
+                        new CultureInfo("en-US")
+                    }
+                };
+                app.UseRequestLocalization(options);
+                app.Run(context =>
+                {
+                    var requestCultureFeature = context.GetFeature<IRequestCultureFeature>();
```

Comment:
Please sync to the latest changes. `context.GetFeature` changed to `context.Features.Get` now.